Firmware/Media/Icons/sample.h

```diff
@@ -0,0 +1,15 @@
+#include <stdint.h>
+
+#include "../../UI/Widgets/Icon.h"
+ 
+struct IconName : public Icon {
+ 
+    uint8_t data[width * height] = {image_data};
```

Comment:
Please mark the placeholders differently, e.g. $WIDTH, so it's visible that the file is only a template and not intended for use.

Reply:
I can't use "$" as it has a different meaning in Makefiles, is it okay if I make it "WIDTH" (in caps)?

Reply:
What about some other special sign, e.g. % -> %WIDTH%

Reply:
yes "%" worked!